From: SeongJae Park <sj@kernel.org>
Date: Sun, 25 May 2025 16:37:53 -0700
Subject: [PATCH] mm/damon/core: fix prototype warning of damon_search()

Prototpye of damon_search() in its kerneldoc comment is having a typo.
Fix it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202505260758.TBfbD1ZO-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index 3f2c8f11210b..41cd947c20d8 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -2966,7 +2966,7 @@ static int __init damon_init(void)
 
 #ifdef DAMON_HACK_IDEA
 /**
- * damos_search() - Return damon_region of a given address and type.
+ * damon_search() - Return damon_region of a given address and type.
  * @addr:	the address of the region to search for.
  * @pid:	PID of the address space.
  *
@@ -2977,7 +2977,7 @@ static int __init damon_init(void)
  *
  * Return: damon_region covering the memory of the given address.
  */
-struct damon_region *damon_search(unsigned long addr)
+struct damon_region *damon_search(unsigned long addr, struct pid *pid)
 {
 	return NULL;
 }
-- 
2.39.5

